doc/user/content/sql/functions/list_agg.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "list_agg Function"
+description: "Concatenates input values (including nulls) into a string. Each value after the
+first is preceded by its corresponding `delimiter`."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+The `list_agg(value, delimiter)` aggregate function concatenates
```

Comment:
There's no delimiter for list_agg. This function is identical to array_agg (not string_agg), except it works on lists instead of arrays.